### PR TITLE
Missing fields in post API

### PR DESCRIPTION
**Pull request by Spartakovic**
*duration
*countries of origin
*year
*series and episode number for episodes

### CODE REVIEW

File: src/main/java/org/atlasapi/input/ItemModelTransformer.java [hunk 1]
```diff
@@ -120,6 +129,10 @@ protected Item setFields(Item item, org.atlasapi.media.entity.simple.Item inputI
             version.setLastUpdated(now);
             version.setManifestedAs(encodings);
         }
+
+        if (!inputItem.getLocations().isEmpty()) {
```
Comment (tfm) [inline src/main/java/org/atlasapi/input/ItemModelTransformer.java]: Why do you need this condition?

File: src/main/java/org/atlasapi/input/ItemModelTransformer.java [hunk 2]
```diff
@@ -141,6 +154,15 @@ protected Item setFields(Item item, org.atlasapi.media.entity.simple.Item inputI
         return item;
     }
 
+    private void setDuration(Version version, org.atlasapi.media.entity.simple.Item inputItem) {
+        for (org.atlasapi.media.entity.simple.Location location : inputItem.getLocations()) {
```
Comment (tfm) [inline src/main/java/org/atlasapi/input/ItemModelTransformer.java]: We're iterating over all `Location`s and using just one? Is there a better way?

File: src/main/java/org/atlasapi/input/ItemModelTransformer.java [hunk 3]
```diff
@@ -66,12 +68,19 @@ protected Item createOutput(org.atlasapi.media.entity.simple.Item inputItem) {
...
     }
 
+    private Item createItem(org.atlasapi.media.entity.simple.Item inputItem) {
+        Item item = new Item();
+        item.setYear(inputItem.getYear());
+        item.setCountriesOfOrigin(inputItem.getCountriesOfOrigin());
+        return item;
+    }
+
     private Item createFilm(org.atlasapi.media.entity.simple.Item inputItem) {
```
Comment (Dalamar42) [inline src/main/java/org/atlasapi/input/ItemModelTransformer.java]: CountriesOfOrigin and Year should be handled for films as well

Comment (Dalamar42) [inline src/main/java/org/atlasapi/input/ItemModelTransformer.java]: Actually, check if you can standardise this so you only have to do it in one place

**Pull request merged**